ui/jobs/jobs.js

```diff
@@ -504,6 +509,13 @@ class Bars {
     return bar;
   }
 
+  playNotification() {
+    console.log('playing sound');
```

Comment:
```suggestion
```

Reply:
Thanks, I need to add a pre-commit hook to look for those...